README.md

```diff
@@ -1,24 +1,166 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 [Blog] <a name="about-project"></a>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+**[OOP: SchooL library]** A fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul> 
+    <li><a href="https://www.ruby-lang.org/en/">RoR</a></li>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/RoR">Linters</a></li>
+
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- posts.
+- Add comments.
+- add likes.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running. To get more information, read the [installation instructions](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md).
+- [RAILS](https://guides.rubyonrails.org/v5.1/getting_started.html) installed and running. To get more information, read the [installation instructions] (https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Rails_guides_lesson.md).
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/hakichris/Blog-App.git
+
+### Install
+
+Install this project with:
+
+```
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```
+  rails s
+```
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Christian Hakizimana**
+
+- GitHub: [@githubhandle](https://github.com/hakichris)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hakichris)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[have a front end]**
+- [ ] **[web application of the program]**
+- [ ] **[responsive for mobile and web]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/hakichris/OOP-School-Library/issues).
```

Comment:
- [ ] Also please consider fixing this link as currently, it redirects users to OOP-School-Library issues. 🤓 🔧

Reply:
fixed, thank you